app/public/css/landing.css

```
@@ -77,10 +121,10 @@
 
 /* Use a neutral themed surface for the inner area so it adapts to light/dark */
 .demo-inner {
-  background: var(--bg-tertiary);
   border-radius: 8px;
   padding: 1rem;
   border: 1px solid var(--falkor-border-tertiary);
+  border: none;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)